<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ballistics Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1321</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="ballisticsForm">
         <property name="labelAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="bcLabel">
           <property name="text">
            <string>BC</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="bcFormLabel">
           <property name="text">
            <string>G1 or G7?</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="coeffSelector">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="g1Selected">
             <property name="text">
              <string>G1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="g7Selected">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>G7</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="veloLabel">
           <property name="text">
            <string>Initial Velocity</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="shLabel">
           <property name="text">
            <string>Sight Height</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="bulletLengthLabel">
           <property name="text">
            <string>Bullet Length</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="bulletGrainsLabel">
           <property name="text">
            <string>Bullet Grains</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="caliberLabel">
           <property name="text">
            <string>Caliber</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="shotAngleLabel">
           <property name="text">
            <string>Shot Angle</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="windSpeedLabel">
           <property name="text">
            <string>Wind Speed</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="windAngleLabel">
           <property name="text">
            <string>Wind Angle</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="twistRateLabel">
           <property name="text">
            <string>Twist Rate</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="zeroDistanceLabel">
           <property name="text">
            <string>Zero Distance</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="temperatureLabel_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Temperature</string>
           </property>
           <property name="setHidden" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="airPressureInHgLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Air Pressure (inHg)</string>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QLabel" name="atmosCheckLabel">
           <property name="text">
            <string>Use Atmosphere Correction?</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QCheckBox" name="useAtmosCorrection">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Yes</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="20" column="0">
          <widget class="QLabel" name="altitudeLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Altitude</string>
           </property>
          </widget>
         </item>
         <item row="20" column="1">
          <widget class="QLineEdit" name="altitude">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="21" column="0">
          <widget class="QLabel" name="humidityLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Humidity</string>
           </property>
          </widget>
         </item>
         <item row="21" column="1">
          <widget class="QLineEdit" name="humidity">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="velo">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="value">
            <number>2800</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="length">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>5.000000000000000</double>
           </property>
           <property name="value">
            <double>1.200000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="sh">
           <property name="maximum">
            <double>5.000000000000000</double>
           </property>
           <property name="value">
            <double>1.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="bc">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.300000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QDoubleSpinBox" name="grains">
           <property name="minimum">
            <double>10.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
           <property name="value">
            <double>168.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QDoubleSpinBox" name="caliber">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>5.000000000000000</double>
           </property>
           <property name="value">
            <double>0.308000000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QDoubleSpinBox" name="angle">
           <property name="minimum">
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QDoubleSpinBox" name="windSpeed">
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QDoubleSpinBox" name="windAngle">
           <property name="minimum">
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDoubleSpinBox" name="twistRate">
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>30.000000000000000</double>
             </property>
             <property name="value">
              <double>8.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="twistInLabel">
             <property name="text">
              <string>: in</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="14" column="1">
          <widget class="QDoubleSpinBox" name="zero">
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QDoubleSpinBox" name="temperature">
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>150.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QDoubleSpinBox" name="inHg">
           <property name="maximum">
            <double>250.000000000000000</double>
           </property>
           <property name="value">
            <double>29.920000000000002</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="calcButton">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="OpenGLShotVisualizer" name="shotVisualizer"/>
       </item>
       <item>
        <widget class="QChartView" name="trajectoryChart"/>
       </item>
       <item>
        <layout class="QFormLayout" name="chartForm">
         <item row="1" column="0">
          <widget class="QLabel" name="viewTypeLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="testLabel">
           <property name="text">
            <string>Step Size</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="stepSize">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="maxYardsLabel">
           <property name="text">
            <string>Max Yards</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="maxYards">
           <property name="minimum">
            <number>50</number>
           </property>
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="toggleViewButton">
           <property name="text">
            <string>Toggle 3D</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
  <customwidget>
   <class>OpenGLShotVisualizer</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">openglshotvisualizer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>useAtmosCorrection</sender>
   <signal>toggled(bool)</signal>
   <receiver>altitudeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useAtmosCorrection</sender>
   <signal>toggled(bool)</signal>
   <receiver>altitude</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useAtmosCorrection</sender>
   <signal>toggled(bool)</signal>
   <receiver>humidity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useAtmosCorrection</sender>
   <signal>toggled(bool)</signal>
   <receiver>humidityLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
